Core/OxidEE_Events.php

```diff
@@ -133,25 +133,22 @@ private static function _extendPaymentMethodTable()
     }
 
     /**
-     * Creates the module's order table
+     * Extends OXID's internal order table with the fields required by the module
      */
-    private static function _createOrderTable()
+    private static function _extendOrderTable()
     {
-        $sQuery = "CREATE TABLE IF NOT EXISTS `wdoxidee_orders` (
-                `wdoxidee_orderid` char(32) character set latin1 collate latin1_general_ci NOT NULL,
-                `wdoxidee_paymentstate` enum('pending','completed','failed','cancelled') NOT NULL DEFAULT 'pending',
-                `wdoxidee_totalordersum` decimal(9,2) NOT NULL,
-                `wdoxidee_capturedamount` decimal(9,2) NOT NULL,
-                `wdoxidee_refundedamount` decimal(9,2) NOT NULL,
-                `wdoxidee_voidedamount` decimal(9,2) NOT NULL,
-                `wdoxidee_currency` varchar(32) NOT NULL,
-                `wdoxidee_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
-                PRIMARY KEY (`wdoxidee_orderid`),
-                KEY `wdoxidee_paymentstate` (`wdoxidee_paymentstate`)
-            ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci";
-
-            $oDb = oxDb::getDb();
-            $oDb->Execute($sQuery);
+        $sQueryState = "ALTER TABLE oxorder ADD COLUMN `WDOXIDEE_PAYMENTSTATE` enum('pending','completed',
+        'failed','cancelled')";
+        self::_addColumnIfNotExists('oxorder', 'WDOXIDEE_PAYMENTSTATE', $sQueryState);
+
+        $sQueryCapAmount = "ALTER TABLE oxorder ADD COLUMN `WDOXIDEE_CAPTUREAMOUNT` decimal(9,2) NOT NULL";
```

Comment:
If i understand your logic correctly i think you missing the authorization amount

Reply:
We removed the amount columns, because we don't use them right now. We will implement them once we need them.